src/ser/mod.rs

```diff
@@ -77,25 +70,18 @@ struct Pretty {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct PrettyConfig {
     /// Limit the pretty-ness up to the given depth.
-    #[serde(default = "default_depth_limit")]
```

Comment:
I'm not sure how this is meant to work, if you remove these serde attributes it will default to 0 or empty strings.

Reply:
You are absolutely right, I forgot to add `#[serde(default)]` (https://serde.rs/container-attrs.html#default) to the struct. Should be fixed now

Reply:
Oh, interesting, did not know about the container-level default attribute.